tests/mmm/test_transformers.py

```diff
@@ -343,6 +344,26 @@ def test_logistic_saturation_min_max_value(self, x, lam):
         assert y_eval.max() <= 1
         assert y_eval.min() >= 0
 
+    def test_inverse_scaled_logistic_saturation_lam_half(self):
+        x = np.array([0.5] * 100)
```

Comment:
Can these be different values then pass as both x and lam in function

Reply:
not sure I understood, do you mean something like this?:

```py
def test_inverse_scaled_logistic_saturation_lam_half(self):
        x = np.array([0.01, 0.1, 0.5, 1, 100])
        y = inverse_scaled_logistic_saturation(x=x, lam=x, eps=np.log(3))
        expected = np.array([0.5] * len(x))
        np.testing.assert_almost_equal(
            y.eval(),
            expected,
            decimal=5,
            err_msg="The function does not behave as expected at eps = ln(3).",
        )
```

Reply:
Yup seems right. Want to check f(x=lam, lam=lam) = 0.5 for various values. Also remove the eps in order to test the default value had this property

Reply:
got it, thanks! just updated the PR